frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -137,19 +138,19 @@ end
 
 function ReaderTypeset:onToggleEmbeddedStyleSheet(toggle)
     self:toggleEmbeddedStyleSheet(toggle)
-    Notification:notify(T( _("Embedded styles are %1."), Notification:getOptionText("ToggleEmbeddedStyleSheet", toggle and 2 or 1)))
+    Notification:notify(T( _("Embedded styles are %1."), OptionsCatalog:getOptionText("ToggleEmbeddedStyleSheet", toggle and 2 or 1)))
```

Comment:
Ideally, the `toggle and 2 or 1` or `mode+1` shouldn't be here, and the mapping should be known to your OptionCatalog, which might build a mapping `event arg (what we get here) => label` instead of `value => label`.